platform-server-rest/src/main/java/uno/cod/platform/server/rest/RestUrls.java

```diff
@@ -20,6 +20,8 @@
     public static final String TASKS = "/tasks";
     public static final String TASKS_ID = "/tasks/{id}";
 
+    public static final String RESULTS_TASKS_SUBMISSIONS = "/results/{resultId}/tasks/{taskId}/submissions";
```

Comment:
why do we submit a submission to a result ? doesn't the result derive from the submission?